modules/datastore/src/Service/Info/ImportInfo.php

```diff
@@ -56,13 +52,13 @@ class ImportInfo {
 
   /**
    * Datastore service.
-   *
-   * @var \Drupal\datastore\DatastoreService
    */
   protected DatastoreService $datastoreService;
 
   /**
    * Constructor.
+   *
+   * @todo Remove the $importServiceFactory argument; it's not used.
```

Comment:
Do we have a follow-up issue?

Reply:
No, pretty low-priority. Do we need to have an issue for every @todo? If so we have a lot of catchup to do :)

Reply:
well, yes, indeed we have a lot of catchup to do. :-) But not a blocker.